### DISCUSSION TITLE
VSCODE tips

### DISCUSSION THREAD

**Discussion by abdullahazmy** [community/community#131857] [Programming Help]
I have created this repository, and I hope some of you can help by adding more useful shortcuts to it.

I was thinking of creating dotfiles within the repository to help people apply preferred settings.

Please feel free to contribute and enhance the repository. Thank you!
[repo-link](https://github.com/abdullahazmy/vscode-helpful)

**Comment by goldieowner17** [comment]
Hey there @abdullahazmy! You'll have more visibility and likely better interaction if you ask this is Programming Help. I've gone ahead and re-categorized this for you.
Cheers!

**Reply by abdullahazmy** [reply]
Noticed, thanks for your help.
